hasher-matcher-actioner/requirements-dev.txt

```diff
@@ -5,4 +5,5 @@ black
 moto
 pandas
 WebTest==2.0.35
-Pillow==8.3.2
\ No newline at end of file
+Pillow==8.3.2
```

Comment:
I conjectured that you asked because if we didn't have the requirement installed, we'd hit a snag.

So, I put the import for hcl2 within a guard and warned anyone trying to use it.